ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -61,6 +61,7 @@ fi
 
 case "${CLUSTER_TYPE}" in
 aws|aws-arm64) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
+aws-china) export AWS_SHARED_CREDENTIALS_FILE=/var/run/aws-china-credential/.awscred;;
```

Comment:
This should not be necessary, the secret should come from the cluster profile and there is no reason why it should be in a different path in the aws-chine cluster profile